packages/common/src/store/ui/withdraw-usdc/slice.ts

```diff
@@ -22,6 +22,9 @@ const slice = createSlice({
     beginWithdrawUSDC: (
       state,
       _action: PayloadAction<{
+        /** Balance in cents. Used for analytics */
+        currentBalance: number
```

Comment:
nit: can't the saga calculate this?

Reply:
It can (and does!) but it happens well into the saga inside a `try` block. I was trying to avoid changing too much about the code I was instrumenting. It also matches the value we will send in all the other events in the components 🤷